src/main/java/Deadlines.java

```diff
@@ -0,0 +1,23 @@
+public class Deadlines extends Task {
+    protected static String type = "[D]";
```

Comment:
Would it be better for the type to be static final and capitalized as in the coding standards for constants since this type won't be changing?

Reply:
Agreed